### DISCUSSION TITLE
Document all possible errors for each endpoint

### DISCUSSION THREAD

**Discussion by Jonny1987** [supabase/supabase#36080] [Feature Requests]
# Improve documentation

## Describe the problem

The API documentation doesn't mention what errors can be returned by each endpoint. For example, The signIn method can return an AuthException with 400 statusCode and either "Email not confirmed" or "Invalid login credentials" as a message for two different errors, but I only discovered these by testing my code, not from reading the documentation. Maybe there are others which I need to know about too. I need to check for these so I can give an appropriate message to users in their own language, so it's no good just displaying the message that supabase returns.

## Describe the improvement

Detail in the documentation which errors are raised by each method in the supabase package.

**Comment by encima** [comment]
Thanks for opening this.

To clarify and refine this request, is it for all APIs under a Project? or specifically for Auth?

**Comment by Jonny1987** [comment]
Look at the docs for flutter (https://supabase.com/docs/reference/dart), I can't see mention of any errors for any section, so I think the answer is: for all APIs.

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
